packages/components/src/unit-control/index.tsx

```diff
@@ -88,7 +88,7 @@ function UnforwardedUnitControl(
 	);
 
 	const [ unit, setUnit ] = useControlledState< string | undefined >(
-		unitProp,
+		units.length === 1 ? units[ 0 ].label : unitProp,
```

Comment:
```suggestion
		units.length === 1 ? units[ 0 ].value : unitProp,
```

Should this not reference the unit's `value` as `label` could be anything? Not sure it differs in most cases for the web, native though uses human readable strings for the labels.

Reply:
Makes sense to me, good catch!